src/features/wallet/root/useActivityItem.tsx

```diff
@@ -63,10 +67,12 @@ const useActivityItem = (
     time: '',
     isFee: false,
     fee: '',
+    feePayer: '',
   })
   const { hntBalanceToDisplayVal } = useCurrency()
   const colors = useColors()
   const { t } = useTranslation()
+  const { makers } = useSelector((state: RootState) => state.heliumData)
```

Comment:
`const makers = useSelector((state: RootState) => state.heliumData.makers)`
Same thing here. Doing it this way makes the equality function only run on `makers` instead of `heliumData`